BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01822-01
+2.0.0-prerelease-01901-01
```

Comment:
In order to call the `OpenCommPort` and not get `UnathorizedAccessException` I need to add extra capabilities to the test runner.
```xml
        <DeviceCapability Name="serialcommunication">
          <Device Id="any">
            <Function Type="name:serialPort" />
          </Device>
        </DeviceCapability>
```